NSTTutorials/src/test/java/com/nst/tutorials/rest/mockgeneration/README.md

```diff
@@ -0,0 +1,17 @@
+[_Return to Tutorials Overview_](https://github.com/eBay/NSTSuite/tree/main/NSTTutorials)
+
+# Mock Generation
```

Comment:
@byarger-ebay Does this seem sufficient? I'm not really sure what else to be adding here for the Mock Generation section